src/cdk-experimental/listbox/listbox.ts

```diff
@@ -89,19 +93,58 @@ export class CdkOption implements ListKeyManagerOption, Highlightable {
     this._active = false;
   }
 
+  select() {
+    if (!this.selected) {
+      this.selected = true;
+      this._emitSelectionChange();
+    }
+  }
+
+  deselect() {
+    if (this.selected) {
+      this.selected = false;
+      this._emitSelectionChange();
+    }
+  }
+
+  /** Applies focus to the option. */
+  focus() {
+    this._elementRef.nativeElement.focus();
+  }
+
   /** Returns true if the option or listbox are disabled, and false otherwise. */
   _isInteractionDisabled(): boolean {
     return (this.listbox.disabled || this._disabled);
   }
 
+  private _emitSelectionChange(isUserInput = false) {
+    const changeEvent = {
+      source: this,
+      isUserInput: isUserInput
+    };
+    this.selectionChange.emit(changeEvent);
```

Comment:
```suggestion
    this.selectionChange.emit({
      source: this,
      isUserInput: isUserInput
    });
```
it's fine to inline the object literal

Reply:
Okay will change.